### PR TITLE
store/tikv: make tikv.ErrTiKV* as normal errors instead of dberror

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: AndreMouche

### What problem does this PR solve?
This PR makes `tikv.ErrTiKV*` as normal errors instead of a dberror and we convert the tikv error to dberror in driver.

Part of https://github.com/pingcap/tidb/issues/22513

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

**Review by andylokandy — APPROVED** [review]

**Review by youjiali1995 — APPROVED** [review]

File: store/tikv/error/errcode.go [hunk 1]
```diff
@@ -33,8 +33,6 @@ const (
 
 	CodeTiKVStoreLimit = 9008
 
-	CodeTiKVStaleCommand          = 9010
```
Comment (siddontang) [inline store/tikv/error/errcode.go:36]: seem here we won't use the Error code later? do we need to update the doc?

Comment (AndreMouche) [reply store/tikv/error/errcode.go:36]: NO，here we just remove this code from `tikv-client`, and the code still exists in `tidb`.

**Pull request merged**